app/docker/views/containers/edit/containerController.js

```diff
@@ -311,6 +311,7 @@ angular.module('portainer.docker').controller('ContainerController', [
       return pullImageIfNeeded()
         .then(stopContainerIfNeeded)
         .then(renameContainer)
+        .then(disconnectNetworksFromOldContainer)
```

Comment:
maybe it's time to migrate the EE code here and just use it directly? feels more clean, and also keeps the codebases in sync